test/uk/gov/hmrc/brm/controllers/BirthEventsControllerSpec.scala

```diff
@@ -497,18 +484,6 @@ class BirthEventsControllerSpec
           header(ACCEPT, result).get shouldBe "application/vnd.hmrc.1.0+json"
         }
 
-        "return InternalServerError when GRO throws Exception" in {
-          when(mockAuditConnector.sendEvent(Matchers.any())(Matchers.any(), Matchers.any())).thenReturn(Future.successful(AuditResult.Success))
-          when(MockController.service.groConnector.getReference(Matchers.any())(Matchers.any())).thenReturn(Future.failed(new Exception("")))
-
```

Comment:
My mistake, I removed this thinking it was a duplicate but it's getDetails what's getting tested. I've reinserted the test for getReference.